src/util/noop.js

```diff
@@ -0,0 +1 @@
+export const noop = () => {};
```

Comment:
in the rest of the code base, noop is written inline, just adding this would be inconsistent probably